### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lindavs**
Updated this pages based on our discussions in KC.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,20 +86,46 @@ DevOps Culture| Everyone should know how stuff runs and keep an eye that service
...
+[Network Warriors]() | Network Products & Automation | TBD 
+[Deadlift](https://packet.atlassian.net/secure/RapidBoard.jspa?rapidView=65) | Production Engineering | [API & Network Availability]() 
+[CMR](https://packet.atlassian.net/secure/RapidBoard.jspa?rapidView=66&projectKey=SWE) | Usage & Billing, Business Processes Automation | TBD
+
+__Future Squads__
+
+These are teams that require more focused attention but don't have the critical mass yet for a well functioning team.
 Squad Name    | Main Focus | SLO | Squad Lead
 :-------------|----------- | --- | -------------
-[CMR](https://packet.atlassian.net/secure/RapidBoard.jspa?rapidView=66&projectKey=SWE) | Usage & Billing | TBD | TBD
```
Comment (batmany13) [inline README.md]: For some reason, this removal broke the view.  Let me see if I can clean it up

File: README.md [hunk 2]
```diff
@@ -86,20 +86,46 @@ DevOps Culture| Everyone should know how stuff runs and keep an eye that service
...
 
-__Proposed Future Squads__
+__Our Chapters__ 
 
-These are teams that require more focused attention but don't have the critical mass yet for a well functioning team.
+Responsibilities| Engineering Manager
+:-------------| ------------- 
+Backend/API/Microservices (Software) |Josue
+UX/UI (Software) | Linda
+Infra Architect | Bruce
```
Comment (batmany13) [inline README.md]: We may need to clarify this as we've had questions about this.  So its not confusing.   Also, its something we want to hire for.

**Comment by batmany13** [comment]
Nice work @lindavs !

File: README.md (hunk 2 above)
Comment (lindavs) [reply README.md]: Maybe we can come up with a better name collaboratively tomorrow

**Pull request merged**